core/drand.go

```diff
@@ -174,6 +174,20 @@ func (d *Drand) BeaconLoop() {
 	d.beacon.Loop(DefaultSeed, d.opts.beaconPeriod, catchup)
 }
 
+func (d *Drand) DistKey(context.Context, *drand.DistKeyRequest) (*drand.DistKeyResponse, error) {
+	pt, err := d.store.LoadDistPublic()
+	if err != nil {
+		return nil, errors.New("We could not load the distributed key")
```

Comment:
It's usually better to leave error message concise, without upper case and with a simple prefix indicating from where it comes. more like `drand: could not load dist. key`